rafthttp/transport.go

```diff
@@ -106,6 +111,7 @@ func (t *transport) Handler() http.Handler {
 	mux := http.NewServeMux()
 	mux.Handle(RaftPrefix, pipelineHandler)
 	mux.Handle(RaftStreamPrefix+"/", streamHandler)
+	mux.Handle(HealthPrefix, probing.NewHandler())
```

Comment:
Maybe named ProbePrefix? It is a little weird that the NewHandler from probing serves HealthPrefix.